mycroft/enclosure/hardware/MycroftLed/led_neo_pixel.py

```diff
@@ -0,0 +1,82 @@
+# Copyright 2020 Mycroft AI Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from subprocess import Popen
+
+class Led:
+    """
+    Class to manipulate the LEDs on an SJ201
+    Conforms to the same interface for all Mycroft
+    LED devices 
+    """
+    def __init__(self):
+        self.num_leds = 12  # sj201 has 12
+        self.leds = list(((0,0,0),) * self.num_leds)
+        self.capabilities = {
+                        "num_leds":10,
+                        "led_colors":"MycroftPalette",
+                        "reserved_leds":[10,11],
+                    }
+
+    def get_capabilities(self):
+        return self.capabilities
+
+    def _update_leds(self):
+        cmd = "sudo "
+        cmd += "/home/pi/mycroft-core/mycroft/enclosure/hardware/util/"
+        cmd += "neo_pixel_set_leds.py "
```

Comment:
It really doesn't matter where it is located. It is here because it is an independent executable callable from anywhere. You can move it if it is a concern just make sure whoever calls it is updated as well.